pygmt/src/coupe.py

```diff
@@ -0,0 +1,379 @@
+"""
+coupe - Plot cross-sections of focal mechanisms.
+"""
+
+import numpy as np
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTError, GMTInvalidInput
```

Comment:
```suggestion
from pygmt.exceptions import GMTInvalidInput
```
Not used in your code.